### DISCUSSION TITLE
so 2.3.1 installed, standalone, import, stuff just don't work as expect, appears broken

### DISCUSSION THREAD

**Discussion by JLT032** [Security-Onion-Solutions/securityonion#2574] [Unsupported Versions]
New to security onion but not to IT it is notable how things break or do not work as expected.

case 1: installed as standalone

attempted pcap imports, did not work, it was probably me
perform so-analyst-install
all prior functionality is broken now

case 2 : installed as import

pcap imports after converting from pcapng format to pcap, found to work after use of the imported link
mutliple parts of the soc don't work
so parts of soc are visible in the status but cannot be started

security onion is urnning in a VM with 16GB RAM and 256GB disk on a 4 core vCPU

**Comment by TOoSmOotH** [comment]
2.3.1 is old. Many things have been improved in the latest version and will save you a lot of headaches.

**Reply by JLT032** [reply]
what good is this remark ? it is a typo, i download the latest version, this is 2.3.21
if any more recent version exist let me know

**Reply by TOoSmOotH** [reply]
How was I supposed to know it was a mistype by you? I don't read minds. 2.3.1 was a valid release version for us which is what I assumed based off the data presented. The remark was there have been quite a lot of improvements since 2.3.1 and there was a good chance they would fix your issue.

**Comment by rwwiv** [comment]
1. Make sure when rebooting Security Onion (as occurs after running `so-analyst-install`) that you verify all services are in the "OK" state with `sudo so-status`. It can take a moment (up to several minutes) for everything to start up, during which time the system may appear to not be working.
2. The SOC sidebar does not update to reflect what services are activated, which means several links will not work on an import install. There is an open issue about this (#1643), but we have no timeline in place for the change.

**Reply by JLT032** [reply]
case 1 

so-status shows everthying working on install as standalone
after execute of so-analys-install most services or not started and do not start

case 2

only few services are installed and reported as working
manual starting of zeek and others fail
import of pcap works here but it may be because i was in a hurry it did not work for case 1

**Comment by JLT032** [comment]
reinstalling as standalone again, from the documentation it appears all is peachy when done and it can work with import and hunt and most other functionalities, is there anything i can do wrong which is obvious to securityonion veterans but not to newbie so users ?
